apps/not-arrived/acceptance/features/_happy-path.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+Feature('Not Arrived - Happy path');
+
+const values = {
+  'received': 'yes',
+  'no-letter': 'false',
+  'delivery-date-month': '6',
+  'delivery-date-year': '2017',
```

Comment:
Nope but these days should probably be dynamic else they're gonna fail in the future...